owlbot.py

```diff
@@ -32,6 +32,7 @@
 # Add templated files
 # ----------------------------------------------------------------------------
 templated_files = common.py_library(cov_level=98, microgenerator=True)
+python.py_samples(skip_readmes=True)
 s.move(
     templated_files, excludes=[".coveragerc"]
 )  # the microgenerator has a good coveragerc file
```

Comment:
```suggestion
s.move(
    templated_files, excludes=[".coveragerc"]
)  # the microgenerator has a good coveragerc file

python.py_samples(skip_readmes=True)
```

Reply:
One minor observation, otherwise LGTM. There were no templated sample files added. Is that expected?